src/Value/Color.php

```diff
@@ -72,23 +72,48 @@ public static function parse(ParserState $oParserState, bool $bIgnoreCase = fals
             $oParserState->consumeWhiteSpace();
             $oParserState->consume('(');
 
+            // CSS Color Module Level 4 says that `rgb` and `rgba` are now aliases; likewise `hsl` and `hsla`.
+            // So, attempt to parse with the `a`, and allow for it not being there.
+            switch ($sColorMode) {
+                case 'rgb':
```

Comment:
For the two cases with the fall-through, please let's have a comment explaining that this is intentional, e.g., like this:

```php
// intentional fall-through
```

Reply:
It's common for `case`s with identical handlers to be grouped like this without comment - I've also seen them written on the same line (e.g. `case 'rgba': case 'hsla':`).  I think it would be clunky to have a comment between the `case`s, and make the code less readable.  However, I've added a comment above the paried `case`s to clarify the intent.

Reply:
Thanks! This makes is easy to see that not having the `break` was intentional instead of an oversight.

Reply:
> This is delayed because I need to figure out how to work around [PHP-CS-Fixer/PHP-CS-Fixer#6572](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/issues/6572). [PHP-CS-Fixer/PHP-CS-Fixer#7624](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7624) might offer a solution.

#798 - Unfortunately is seems we need to disable a rule that has become broken.  [One repository](https://github.com/akeneo/pim-community-dev/pull/17431) has even placed an upper version limit on PHP CS Fixer, pending resolution to the breakage.